### PR TITLE
fix: build then launch link

### PR DESCRIPTION
**Pull request by Applenice**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [X] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

fix source codes link from https://github.com/apache/apisix-dashboard/blob/master/docs/en/latest/deploy.md  to https://github.com/apache/apisix-dashboard/blob/master/docs/en/latest/install.md

**Checklist:**

- [X] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]
Good!!

**Review by iamayushdas — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -77,9 +77,9 @@ Password: admin
 
 Support the following ways currently.
 
-- [Source Codes](./docs/en/latest/deploy.md)
+- [Source Codes](./docs/en/latest/install.md#source-source)
 - [Docker](./docs/en/latest/deploy-with-docker.md)
-- [RPM package（only for CentOS 7）](./docs/en/latest/deploy-with-rpm.md)
+- [RPM package（only for CentOS 7）](./docs/en/latest/install.md#rpm-rpm)
```
Comment (bzp2010) [inline README.md:82]:
```suggestion
- [RPM, Docker, Source Codes](./docs/en/latest/install.md)
- [Rebuild docker image](./docs/en/latest/deploy-with-docker.md)
```

**Review by bzp2010 — COMMENTED** [review]
This looks a little clearer, do you think?

File: README.md (hunk 1 above)
Comment (bzp2010) [inline README.md:82]:
```suggestion
- [RPM, Docker, Source Codes](./docs/en/latest/install.md)
- [Rebuild docker image](./docs/en/latest/deploy-with-docker.md)
```

Comment (Applenice) [reply README.md:82]: Agree, great😄Would it be better to swap the position of Docker and RPM and keep the order of the file titles？

Comment (bzp2010) [reply README.md:82]: Sure, that's my mistake, I will change the suggested content.

Comment (Applenice) [reply README.md:82]: I show here: `Applying suggestions on deleted lines is currently not supported.`

Comment (Applenice) [reply README.md:82]: @juzhiyuan @bzp2010 emmm, I can't Commit suggestion now😐Can you operate it?

Comment (juzhiyuan) [reply README.md:82]: image

The seem warning 🤔 Could you update this PR manually?

Comment (Applenice) [reply README.md]: 😮I have updated once, you can see it?

**Review by bzp2010 — APPROVED** [review]
LGTM

**Comment by Applenice** [comment]
Hi, can this merge?

**Comment by juzhiyuan** [comment]
Yes, let me rerun the failed tests first.

**Comment by juzhiyuan** [comment]
🤔 Why it fail?? @LiteSun please take a look.

**Comment by bzp2010** [comment]
> 🤔 Why it fail?? @LiteSun please take a look.

@juzhiyuan This is the case of instability in the test, and it is usually sufficient to perform several retries.

**Review by 1502shivam-singh — APPROVED** [review]

**Pull request merged**